tests/integration/logging/monolog3/test_monolog_context_filter_rule7.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+Test that Monolog3 instrumentation filters context data:
+Rule 7: 
+   include = "A*"
+   exclude = ""
+   input = "A" "B" "C"
+   expect = "A"
+*/
+
+/*SKIPIF
+<?php
+
+require('skipif.inc');
+
+*/
+
+/*INI
+newrelic.application_logging.enabled = true
+newrelic.application_logging.forwarding.enabled = true
+newrelic.application_logging.metrics.enabled = true
+newrelic.application_logging.forwarding.max_samples_stored = 10
+newrelic.application_logging.forwarding.log_level = DEBUG
+newrelic.application_logging.forwarding.context_data.enabled = 1
+newrelic.application_logging.forwarding.context_data.include = "A*"
+newrelic.application_logging.forwarding.context_data.exclude = ""
```

Comment:
Is there a test where `newrelic.application_logging.forwarding.context_data.exclude` is not defined in INI?

Reply:
I modified `test_monolog_context_simple.php` to only enabled context attributes but does not specify any include or exclude rules.

Commit [fc11c62](https://github.com/newrelic/newrelic-php-agent/commit/fc11c620ee981dde9ac312c02e163f96db025eb3).